### PR TITLE
feat: Bot API 7.0

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots/api#december-29-2023

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]

File: methods.ts [hunk 1]
```diff
@@ -67,7 +70,7 @@ export type ApiMethods<F> = {
...
     timeout?: number;
-    /** A list of the update types you want your bot to receive. For example, specify [“message”, “edited_channel_post”, “callback_query”] to only receive updates of these types. See Update for a complete list of available update types. Specify an empty list to receive all update types except chat_member (default). If not specified, the previous setting will be used.
+    /** A list of the update types you want your bot to receive. For example, specify ["message", "edited_channel_post", "callback_query"] to only receive updates of these types. See Update for a complete list of available update types. Specify an empty list to receive all update types except chat_member, message_reaction, and message_reaction_count (default). If not specified, the previous setting will be used.
 
     Please note that this parameter doesn't affect updates created before the call to the getUpdates, so unwanted updates may be received for a short period of time. */
     allowed_updates?: ReadonlyArray<Exclude<keyof Update, "update_id">>;
```
Comment (roziscoding) [inline methods.ts:76]: question (non-blocking): can't we use the available updates type here?

**Review by roziscoding — APPROVED** [review]
LGTM

File: methods.ts (hunk 1 above)
Comment (KnorpelSenf) [reply methods.ts:76]: Those from the core lib?

Comment (roziscoding) [reply methods.ts:76]: oh, I got confused, I thought I was looking at the core lib 🤪

**Pull request merged**